src/app/client/src/app/modules/player-helper/components/content-credits/content-credits.component.ts

```diff
@@ -35,12 +36,18 @@ export class ContentCreditsComponent implements OnInit, OnChanges {
   *@param {ResourceService} SearchService Reference of SearchService
   *@param {WorkSpaceService} WorkSpaceService Reference of SearchService
   */
+showCopiedContent = false;
+ instance: string;
 
   constructor(resourceService: ResourceService) {
     this.resourceService = resourceService;
   }
 
   ngOnInit() {
+    this.instance = _.upperCase(this.resourceService.instance);
+    if (window.location.href.toString().includes('course')) {
```

Comment:
This is not the correct way to check if this is a course or not please check if we have an alternative